website/docs/12.x/docs/api/render.mdx

```diff
@@ -13,7 +13,7 @@ The `render` function is the entry point for writing React Native Testing Librar
 import { render } from '@testing-library/react-native';
 
 test('basic test', () => {
-  render(<MyApp />);
+  const screen = render(<MyApp />);
```

Comment:
Instead of capturing `render` output we recommend using `screen` object exported by RNTL, so rather than update this line, we should add `screen` to the import above.